lib/routes/surveyAdmin.js

```diff
@@ -0,0 +1,38 @@
+Router.route('/survey/admin', {
+  name: 'survey.admin',
+
+  waitOn() {
+    return [Meteor.subscribe('teams'), Meteor.subscribe('users'), Meteor.subscribe('survey')];
+  },
+
+  action() {
+    Session.set('title', 'SEMS | Admin');
+    this.render('surveyAdmin');
+  },
+
+});
+
+Router.route('/survey/admin/:slug/', {
+  name: 'admin.teamInfo',
+  layoutTemplate: 'teamInfoAdmin',
+
+  waitOn() {
+    return [Meteor.subscribe('teams'), Meteor.subscribe('fullUsers')];
+  },
+
+  action() {
+    var team = Teams.findOne({ slug: this.params.slug });
+
+    if (!team)
+      this.redirect('survey.admin');
+
+
+    Session.set('title', `${team.name}`);
```

Comment:
Ummmm, not sure, I mean all admin views we have are not stating the specific thing I am editing, you may prefix it with admin though if you need it like that to be different than the team view